src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/ExtractVariantAnnotations.java

```diff
@@ -0,0 +1,361 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr.scalable;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.apache.commons.lang3.tuple.Triple;
+import org.apache.commons.math3.random.RandomGenerator;
+import org.apache.commons.math3.random.RandomGeneratorFactory;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.copynumber.utils.HDF5Utils;
+import org.broadinstitute.hellbender.tools.walkers.vqsr.VariantRecalibrator;
+import org.broadinstitute.hellbender.tools.walkers.vqsr.scalable.data.LabeledVariantAnnotationsData;
+import org.broadinstitute.hellbender.tools.walkers.vqsr.scalable.data.VariantType;
+import picard.cmdline.programgroups.VariantFilteringProgramGroup;
+
+import java.io.File;
+import java.util.List;
+import java.util.Random;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+/**
+ * Extracts site-level variant annotations, labels, and other metadata from a VCF file to HDF5 files.
+ *
+ * <p>
+ *     This tool is intended to be used as the first step in a variant-filtering workflow that supersedes the
+ *     {@link VariantRecalibrator} workflow. This tool extracts site-level annotations, labels, and other relevant metadata
+ *     from variant sites (or alleles, in allele-specific mode) that are or are not present in specified labeled
+ *     resource VCFs (e.g., training or calibration VCFs). The former, present sites are considered labeled; each site
+ *     can have multiple labels. The latter sites are considered unlabeled and can be randomly downsampled using
```

Comment:
Included this line in the above tweak---note that the original "latter" was referring to the sites that "are not present in resource VCFs," rather than the calibration sites. Confusing, indeed!